server/channels/app/plugin_api_test.go

```diff
@@ -304,7 +304,7 @@ func TestPluginAPIUpdateUserPreferences(t *testing.T) {
 }
 
 func TestPluginAPIGetUsers(t *testing.T) {
-	th := Setup(t)
+	th := Setup(t).DeleteBots()
```

Comment:
It seems slightly weird to scatter `DeleteBots()` on a handful of tests. Should we just use `SkipProductsInitialization` in this case to better capture the intent?